### PR TITLE
initial commit argocd-bootstrap

### PR DESCRIPTION
**Pull request by gnekic**
Add a directory that contains a set of manifests to install the argocd-operator and argocd, with accompanying README. I would love to add a sample app to be managed by argocd in future work

### CODE REVIEW

File: argocd-bootstrap/3-argocd/argocd-basic.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+kind: ArgoCD
+metadata:
+  annotations:
+    example.com/managed-by: gitops
+    example.com/scm-url: git@github.com:redhat-cop/declarative-openshift.git
+  labels:
+    example: basic
+    example.com/component: operators
+    example.com/project: argocd-bootstrap
+  name: example-argocd
```
Comment (sabre1041) [inline argocd-bootstrap/3-argocd/argocd-basic.yaml:11]: `namespace` field should be specified

File: argocd-bootstrap/3-argocd/argocd-basic.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+spec:
+  server:
+    route: true
```
Comment (sabre1041) [inline argocd-bootstrap/3-argocd/argocd-basic.yaml:14]: My guess is that you would not want this to be present in the initial manifest as to add it as part of the exercise

**Comment by etsauer** [comment]
@sabre1041 I'd like to separate the handling of the operator race condition as a separate issue. #14

File: argocd-bootstrap/3-argocd/argocd-basic.yaml (hunk 1 above)
Comment (gnekic) [reply argocd-bootstrap/3-argocd/argocd-basic.yaml:11]: addressed in latest comit

**Pull request merged**